worlds/sc2/MissionOrders.py

```diff
@@ -0,0 +1,389 @@
+        ],
+        SC2Campaign.LOTV: [
+            FillMission(MissionPools.STARTER, [MissionConnection(-1, SC2Campaign.LOTV)], "Aiur",completion_critical=True),
+            FillMission(MissionPools.EASY, [MissionConnection(0, SC2Campaign.LOTV)], "Aiur", completion_critical=True),
+            FillMission(MissionPools.EASY, [MissionConnection(1, SC2Campaign.LOTV)], "Korhal", completion_critical=True),
+            FillMission(MissionPools.MEDIUM, [MissionConnection(1, SC2Campaign.LOTV)], "Shakuras", completion_critical=True),
+            FillMission(MissionPools.MEDIUM, [MissionConnection(2, SC2Campaign.LOTV), MissionConnection(3, SC2Campaign.LOTV)], "Purifier", completion_critical=True),
+            FillMission(MissionPools.HARD, [MissionConnection(6, SC2Campaign.LOTV)], "Purifier", completion_critical=True),
+            FillMission(MissionPools.HARD, [MissionConnection(4, SC2Campaign.LOTV)], "Ulnar", completion_critical=True),
+            FillMission(MissionPools.HARD, [MissionConnection(6, SC2Campaign.LOTV)], "Tal'darim", completion_critical=True),
+            FillMission(MissionPools.HARD, [MissionConnection(5, SC2Campaign.LOTV), MissionConnection(7, SC2Campaign.LOTV)], "Return to Aiur", completion_critical=True),
+            FillMission(MissionPools.FINAL, [MissionConnection(8, SC2Campaign.LOTV)], "Return to Aiur", completion_critical=True),
+        ],
+        SC2Campaign.EPILOGUE: [
+            FillMission(MissionPools.VERY_HARD, [MissionConnection(12, SC2Campaign.WOL), MissionConnection(12, SC2Campaign.HOTS), MissionConnection(9, SC2Campaign.LOTV)], "_1", completion_critical=True),
+            FillMission(MissionPools.FINAL, [MissionConnection(0, SC2Campaign.EPILOGUE)], "_2", completion_critical=True),
+        ],
+        SC2Campaign.NCO: [
+            FillMission(MissionPools.EASY, [MissionConnection(-1, SC2Campaign.NCO)], "_1", completion_critical=True),
+            FillMission(MissionPools.MEDIUM, [MissionConnection(0, SC2Campaign.NCO)], "_1", completion_critical=True),
+            FillMission(MissionPools.MEDIUM, [MissionConnection(1, SC2Campaign.NCO)], "_2", completion_critical=True),
+            FillMission(MissionPools.HARD, [MissionConnection(2, SC2Campaign.NCO)], "_3", completion_critical=True),
+            FillMission(MissionPools.FINAL, [MissionConnection(3, SC2Campaign.NCO)], "_3", completion_critical=True),
+        ]
+    }
+
+
+smooth_difficulty = [MissionPools.EASY,
+                     MissionPools.MEDIUM, MissionPools.MEDIUM,
+                     MissionPools.HARD, MissionPools.HARD,
+                     MissionPools.VERY_HARD]
+max_difficulty = len(smooth_difficulty) - 1
+
+
+def make_golden_path(world: World, num_missions: int) -> List[FillMission]:
+    chain_name_options = ['Mar Sara', 'Char', 'Kaldir', 'Zerus', 'Skygeirr Station',
+                   'Dominion Space', 'Korhal', 'Aiur', 'Shakuras', 'Ulnar']
```

Comment:
(kind of for fun, kind of because I think people will see a lot of these pretty quickly) recommend adding Umoja, Moria, Tarsonis. For less-traveled planets or BW throwbacks, maybe Braxis, Typhon XI, Agria, Chau Sara, Antiga Prime, Bel'Shir, Redstone, Meinhoff, Tyrador, Xil, Zhakul. If you add enough, you can slip in a Magnemania as an easter egg and I won't be the one to stop you.

Reply:
How about "Azeroth"?

Reply:
Added some more names.

Reply:
I'd pick rather places actually associated with SC universe and Chau Sara is the place where Loomings campaign takes place,
You could also add Valhalla (from Engine of Destruction), Haven (Haven's Fall / Safe Haven)

Reply:
I think a semi-joking reference to another Blizzard game is pretty cool. I'm in support of Azeroth and Sanctuary myself, I think the limit should be either one non-starcraft planet or one per other major blizzard property. Azeroth + Sanctuary + Draenor + Crouton might be a little much (idk where Crouton is from?). Haven and Valhalla would be cool but not necessary.

In other words, I'm happy to call this resolved myself.

Reply:
Crouton is a reference to The Lost Vikings.  The alien villain is from the Croutonian Empire (there's a salad theme to the names).  I thought it'd be cute to have an obscure reference to one of the oldest Blizzard IPs.  And there are Hydralisks on Azeroth..

I'd rather not add new chain names that can be confused for other chain names, like Chau Sara/Mar Sara, or mission names, like Haven/Safe Haven/Haven's Fall.  I want to add region hinting in a PR after this, and that would hurt the readability of a long list of hints (Zealot is at For Aiur!: Southwest Hive at Haven).  Valhalla would be fine, though.

Reply:
I've merged in the new PR and added Ziktofel's suggested chain names to golden path.